### PR TITLE
Fixed minor typos and misc formatting

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: templates/DatasetReview/file-submit.html.twig [hunk 1]
```diff
@@ -24,10 +24,11 @@
             {% set sftpuser=app.user.isPosix() %}
             <div id="tabs-2"> <!-- Upload via SFTP/Ingest-->
                 <p>Dataset files that are too large via the normal Upload mechanism can be uploaded to the griidc-ingest
-                    server using Globus or SFTP and then the folder selected below.  See your user profile page (link)
+                    server using Globus or SFTP and then the folder selected below.  See your
+                    <a href="{{ path('pelagos_app_ui_person_profile_page', {'id': app.user.id}) }}' }}">user profile page</a>
```
Comment (fightingtexasaggie) [inline templates/DatasetReview/file-submit.html.twig:28]: extra }}'

File: templates/DatasetSubmission/file-submit.html.twig [hunk 2]
```diff
@@ -34,10 +34,11 @@
             {% set sftpuser=app.user.isPosix() %}
             <div id="tabs-2"> <!-- Upload via SFTP/Ingest-->
                 <p>Dataset files that are too large via the normal Upload mechanism can be uploaded to the griidc-ingest
-                    server using Globus or SFTP and then the folder selected below.  See your user profile page (link)
+                    server using Globus or SFTP and then the folder selected below.  See your
+                    <a href="{{ path('pelagos_app_ui_person_profile_page', {'id': app.user.id}) }}' }}">user profile page</a>
```
Comment (fightingtexasaggie) [inline templates/DatasetSubmission/file-submit.html.twig:38]: same as above

**Review by mickel1138 — APPROVED** [review]
LGTM!

File: templates/DatasetReview/file-submit.html.twig [hunk 3]
```diff
@@ -25,7 +25,7 @@
             <div id="tabs-2"> <!-- Upload via SFTP/Ingest-->
                 <p>Dataset files that are too large via the normal Upload mechanism can be uploaded to the griidc-ingest
                     server using Globus or SFTP and then the folder selected below.  See your
-                    <a href="{{ path('pelagos_app_ui_person_profile_page', {'id': app.user.id}) }}' }}">user profile page</a>
+                    <a href="{{ path('pelagos_app_ui_person_profile_page', {'id': app.user.id}) }}">user profile page</a>
```
Comment (fightingtexasaggie) [inline templates/DatasetReview/file-submit.html.twig:28]:
```suggestion
<a href="{{ path('pelagos_app_ui_person_profile_page', {'id': app.user.id}) }}" TARGET="_NEW">user profile page</a>
```

File: templates/DatasetSubmission/file-submit.html.twig [hunk 4]
```diff
@@ -35,7 +35,7 @@
             <div id="tabs-2"> <!-- Upload via SFTP/Ingest-->
                 <p>Dataset files that are too large via the normal Upload mechanism can be uploaded to the griidc-ingest
                     server using Globus or SFTP and then the folder selected below.  See your
-                    <a href="{{ path('pelagos_app_ui_person_profile_page', {'id': app.user.id}) }}' }}">user profile page</a>
+                    <a href="{{ path('pelagos_app_ui_person_profile_page', {'id': app.user.id}) }}">user profile page</a>
```
Comment (fightingtexasaggie) [inline templates/DatasetSubmission/file-submit.html.twig:38]:
```suggestion
<a href="{{ path('pelagos_app_ui_person_profile_page', {'id': app.user.id}) }}" TARGET="_NEW">user profile page</a>
```

**Review by fightingtexasaggie — APPROVED** [review]

**Pull request merged**